README.md

```
@@ -87,3 +87,121 @@
 ### 추천 상품 장바구니 추가 테스트
 - [x] 추천 상품을 장바구니에 추가할 수 있다.
 - [x] 추천 상품 추가 후 장바구니 상태가 갱신된다.
+
+---
+
+## 🚀 3단계 - Navigation
+
+## 🎯 기능 목록
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)